sample/SampleAspNetCoreApp/SampleAspNetCoreApp.csproj

```diff
@@ -20,4 +31,21 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Elastic.Apm.NetCoreAll\Elastic.Apm.NetCoreAll.csproj" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
We had a discussion about this here: https://github.com/elastic/apm-agent-dotnet/pull/622#discussion_r354009316 

I ran into that problem on netcoreapp2.x and adding this back solved the problem. It's still strange to me that we need this, but I'd suggest keeping it, otherwise we seem to run into the problem mentioned in https://github.com/elastic/apm-agent-dotnet/pull/622#discussion_r355208499.